plugins/manager/assets/widget.js

```diff
@@ -147,6 +147,12 @@ $(document).on('rex:ready',function() {
                     let multiple = this.dataset.multiple;
                     let value = this.dataset.value;
 
+                    var event = opener.jQuery.Event('rex:YForm_selectData')
+                    opener.jQuery(window).trigger(event, [id, value, multiple])
+                    if (!event.isDefaultPrevented()) {
+                      self.close()
+                    }
+                    
```

Comment:
Vielen Dank schon mal. :) Kannst du das ohne jQuery umsetzen?

Reply:
Vermutlich schon, aber weshalb denn? :) wird ja ohnehin praktisch überall noch jQuery verwendet in diesem Skript.

Reply:
weil ich das bewusst ohne jquery alles im manager umgesetzt habe, damit wir irgendwann redaxo updaten können und yform davon nicht abhängig ist.